### PR TITLE
feat create profile form

### PR DESCRIPTION
**Pull request by nurba710**

### CODE REVIEW

File: src/components/createProfileForm/CreateProfileForm.tsx [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+import { ProfileTypes } from './Types'
+import Input from '../Input/Input'
+import Textarea from '../Textarea/Textarea'
+import Button from '../Button/Button'
+import { useProfilePOSTMutation } from '../../service/ProfileService'
+
+const CreateProfileForm = () => {
+	const [profile] = useProfilePOSTMutation()
+	const handleSubmit = (values: ProfileTypes) => {
+		console.log(values, 'values')
```
Comment (Nurreles) [inline src/components/createProfileForm/CreateProfileForm.tsx:25]: delete console log

**Review by Nurreles — APPROVED** [review]

File: src/components/createProfileForm/CreateProfileForm.tsx (hunk 1 above)
Comment (Ruslan1516) [inline src/components/createProfileForm/CreateProfileForm.tsx:25]: remove console

File: src/components/createProfileForm/CreateProfileForm.tsx [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+		profile(values).unwrap()
+	}
+
+	return (
+		<Formik
+			initialValues={{
```
Comment (Ruslan1516) [inline src/components/createProfileForm/CreateProfileForm.tsx:31]: move initial values to other file

File: src/components/createProfileForm/CreateProfileForm.tsx [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+							<Input
+								onChange={handleChange}
+								name={'yearsOfWorkExp'}
+								value={values.yearsOfWorkExp}
+								type={'text'}
+								width={'200px'}
+							/>
+							<ProfileError>{yearsOfWorkExp}</ProfileError>
+						</ProfileInputContainer>
+						<FieldArray name={'skills'}>
```
Comment (Ruslan1516) [inline src/components/createProfileForm/CreateProfileForm.tsx:127]: i think this part of the form, can be moved to another component

File: src/components/createProfileForm/CreateProfileForm.tsx [hunk 4]
```diff
@@ -0,0 +1,207 @@
...
+							{({ push, remove }) => (
+								<React.Fragment>
+									<ProfileFormTitle>Навыки</ProfileFormTitle>
+									{values.skills.map((skill, index) => (
+										<ProfileSkillContainer key={index}>
```
Comment (Ruslan1516) [inline src/components/createProfileForm/CreateProfileForm.tsx:132]: can skills be undefined?

File: src/components/createProfileForm/CreateProfileForm.tsx [hunk 5]
```diff
@@ -0,0 +1,207 @@
...
+												name={'Удалить'}
+												onClick={() => remove(index)}
+											/>
+										</ProfileSkillContainer>
+									))}
+									<Button
+										margin={'0 0 20px 0'}
+										name={'Добавить навык'}
+										type={'button'}
+										onClick={() =>
```
Comment (Ruslan1516) [inline src/components/createProfileForm/CreateProfileForm.tsx:171]: move "onclick" to another function

File: src/components/createProfileForm/CreateProfileForm.tsx [hunk 6]
```diff
@@ -0,0 +1,207 @@
...
+												grade: 0,
+											})
+										}
+									/>
+								</React.Fragment>
+							)}
+						</FieldArray>
+						<ProfileInputContainer>
+							<ProfileInputTitle htmlFor={'avatar'}>
+								Фотография для профиля
```
Comment (Ruslan1516) [inline src/components/createProfileForm/CreateProfileForm.tsx:183]: i think, we can create component for ProfileInputContainer

**Review by Ruslan1516 — APPROVED** [review]

**Review by Ruslan1516 — APPROVED** [review]

**Pull request merged**